public/app/percona/check/components/AllChecksTab/CheckTableRow.test.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { ReactWrapper, mount } from 'enzyme';
 import { CheckService } from 'app/percona/check/Check.service';
+import { LoaderButton } from '@percona/platform-core';
 import { CheckTableRow } from './CheckTableRow';
 import { Messages } from './AllChecksTab.messages';
-import { LoaderButton } from '@percona/platform-core';
-
 
+jest.mock('shared/components/helpers/notification-manager');
```

Comment:
Please remove this line. This helper doesn't exist anymore. Use app events directly if needed.

Reply:
Was there while in draft status, removed